src/bag3_digital/layout/stdcells/memory.py

```diff
@@ -612,6 +612,7 @@ def draw_layout(self) -> None:
             seg = seg_dict['nor']
             seg_t1 = seg_dict['keep']
             seg_t0 = seg_dict['in']
+        print(f'nor:{seg}, keep: {seg_t1}, in: {seg_t0}')
```

Comment:
Either remove print statement, or make it more descriptive (mention the name of the generator class, and the fact that these are number of fingers). Also, add missing space between "nor:" and "{seg}".